[docs][python] add conda-forge install instructions (#3544)
microsoft/LightGBM: python-package/README.rst

It is **strongly not recommended** to use this version of LightGBM!

> Install from `conda-forge <https://anaconda.org/conda-forge/lightgbm>`_ Using ``conda-forge``

Reviewer: ```suggestion
Install from `conda-forge <https://anaconda.org/conda-forge/lightgbm>`_
```

> Install from `conda-forge <https://anaconda.org/conda-forge/lightgbm>`_
> '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

Reviewer 2: ```suggestion
Install from `conda-forge channel <https://anaconda.org/conda-forge/lightgbm>`_ Using ``conda``
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```

Reviewer: @StrikerRUS I previously recommended removing the "Using" part (https://github.com/microsoft/LightGBM/pull/3544#discussion_r554287324). I think it makes the title more verbose but doesn't provide any additional information

Reviewer 2: Ah, I see, OK!

Then please ignore my previous suggestion. And I think it will make sense to remove the "Using" part from `Install from PyPI Using pip` heading above.

Author: Thanks for the discussion. I'm unsure on what the suggestion is here. You are welcome to push to my branch.

Reviewer: sorry for the confusion. I just pushed https://github.com/microsoft/LightGBM/pull/3544/commits/5a91ca1a7df340c383f0d588cefb451c7b17f0a7 to resolve this thread.

Reviewer: and https://github.com/microsoft/LightGBM/pull/3544/commits/ffd78bdb3a36a839215abbfb6d19e8f48876e0dd to address @StrikerRUS 's suggestion.

> If you use ``conda`` to manage Python dependencies, you can install ``lightgbm`` using ``conda install``.

Reviewer 2: For the consistency with the rest of this document.
```suggestion
If you use ``conda`` to manage Python dependencies, you can install LightGBM using ``conda install``.
```

> **Note**: The `lightgbm conda-forge package <https://github.com/conda-forge/lightgbm-feedstock>`_.  is not maintained by LightGBM maintainers.

Reviewer 2: ```suggestion
**Note**: The `lightgbm conda-forge feedstock <https://github.com/conda-forge/lightgbm-feedstock>`_ is not maintained by LightGBM maintainers.
```

> You may need to install python via `anaconda/miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ first.
> 
> .. code:: sh
> 
>     conda install -c conda-forge lightgbm
>     
> **Note**: The `lightgbm conda-forge package <https://github.com/conda-forge/lightgbm-feedstock>`_.  is not maintained by LightGBM maintainers.

Reviewer: ```suggestion
If you use ``conda`` to manage Python dependencies, you can install ``lightgbm`` using ``conda install``.

**Note**: The `lightgbm conda-forge package <https://github.com/conda-forge/lightgbm-feedstock>`_.  is not maintained by LightGBM maintainers.

.. code:: sh

    conda install -c conda-forge lightgbm
```

Can you please use this rewording?
